onebusaway-android/src/androidTest/java/org/onebusaway/android/nav/test/NavigationTest.java

```diff
@@ -792,7 +844,7 @@ public int getAutomatedGetReadyIndex() {
         void runSimulation(int expectedGetReadyIndex, int expectedPullCordIndex) {
             NavigationServiceProvider provider = new NavigationServiceProvider(mTripId,
                     mDestinationId);
-
+            Location mPrevLocation = null; // variable to store the location to check for duplicates
```

Comment:
Could you please change this to `prevLocation`?  The `m` prefix is only for class member variables.

Reply:
Also, you can remove the comment here.  Typically I don't like comments on the same line as code, and I think the use of this is obvious enough we don't need a comment.